### PR TITLE
Feat/sequences api

### PR DESCRIPTION
**Pull request by polomani**

### CODE REVIEW

File: src/types/types.ts [hunk 1]
```diff
@@ -167,8 +170,101 @@ export interface Asset extends ExternalAsset, AssetInternalId {
...
+
+/**
+ * Describes a new column
+ */
+export interface ExternalSequenceColumn
+  extends ExternalSequenceColumnBase,
+    ExternalId {}
+
+interface ExternalSequenceColumnBase {
+  name?: ColumnName;
```
Comment (ghost) [inline src/types/types.ts]: Maybe `SequenceColumnName` and `SequenceColumnDescription`?

File: src/types/types.ts [hunk 2]
```diff
@@ -510,10 +635,23 @@ export type EXECUTE = 'EXECUTE';
 
 export type EventChange = EventChangeById | EventChangeByExternalId;
 
+export type SequenceChange = SequencePatch & IdEither;
```
Comment (ghost) [inline src/types/types.ts:638]: Why not interface that extends? More consistent with other similar types

File: src/types/types.ts [hunk 3]
```diff
@@ -724,6 +862,36 @@ export interface Filter {
...
+   * Columns to be included. Specified as list of column externalIds.
+   * In case this filter is not set, all available columns will be returned.
+   */
+  columns?: string[];
+}
+
+/**
+ * A request for datapoints stored
+ */
+export type SequenceRowsRetrieve = SequenceRowsRetriveData & IdEither;
```
Comment (ghost) [inline src/types/types.ts:886]: Same here. Interface that extends

File: src/resources/baseResourceApi.ts [hunk 4]
```diff
@@ -302,3 +313,11 @@ export abstract class BaseResourceAPI<
 type ListEndpoint<QueryType extends FilterQuery, WrapperType> = (
   query?: QueryType
 ) => Promise<HttpResponse<CursorResponse<WrapperType>>>;
+
+interface PostInParallelWithAutomaticChunkingParams<RequestType, ParamsType> {
```
Comment (ghost) [inline src/resources/baseResourceApi.ts:317]: 😆 ILikeVeryLongTypeNamesThatAreUnderstandableAndPrecise

**Review by ghost — CHANGES_REQUESTED** [review]
Very good 🚀

File: src/types/types.ts (hunk 3 above)
Comment (polomani) [reply src/types/types.ts:886]: IdEither is a union type, when you do `SequenceRowsRetriveData & IdEither` it actually creates type, `(SequenceRowsRetriveData & InternalId) | (SequenceRowsRetriveData & ExternalId)`
How will you do it with extends? You cannot extend type

File: src/types/types.ts (hunk 2 above)
Comment (polomani) [reply src/types/types.ts:638]: answered below

File: src/resources/sequences/sequencesApi.ts [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+  };
+
+  /**
+   * [Delete sequences](https://docs.cognite.com/api/v1/#operation/deleteSequences)
+   *
+   * ```js
+   * await client.sequences.delete([{id: 123}, {externalId: 'abc'}]);
+   * ```
+   */
+  public delete = (ids: IdEither[]) => {
```
Comment (ghost) [inline src/resources/sequences/sequencesApi.ts]: Missing explicit return type

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**